Modules/Microsoft365DSC/DSCResources/MSFT_IntuneWindowUpdateForBusinessRingUpdateProfileWindows10/settings.json

```diff
@@ -0,0 +1,39 @@
+{
+    "resourceName": "IntuneWindowUpdateForBusinessRingUpdateProfileWindows10",
+    "description": "This resource configures an Intune Window Update For Business Ring Update Profile for Windows10.",
+    "permissions":    {
+    "graph":  {
+                  "delegated":  {
+                                    "read":  [
+                                                 {
+                                                     "name":  "DeviceManagementConfiguration.Read.All"
+                                                 },
+                                                 {
+                                                     "name":  "DeviceManagementConfiguration.ReadWrite.All"
+                                                 }
+                                             ],
+                                    "update":  [
+                                                   {
+                                                       "name":  "DeviceManagementConfiguration.ReadWrite.All"
+                                                   }
+                                               ]
+                                },
+                  "application":  {
+                                      "read":  [
+                                                   {
+                                                       "name":  "DeviceManagementConfiguration.Read.All"
+                                                   },
+                                                   {
+                                                       "name":  "DeviceManagementConfiguration.ReadWrite.All"
```

Comment:
The permissions are generated from Find-MgGraphCommand and are those used in the doc
I agree with you write is probably not needed - do you want me to modify that or follow the doc? 
![image](https://user-images.githubusercontent.com/79221284/216932885-76d63864-dfff-4764-8753-51496a7ab516.png)
https://learn.microsoft.com/en-us/graph/api/intune-softwareupdate-windowsfeatureupdateprofile-get?view=graph-rest-beta

Reply:
This documentation lists all permissions that can be set. It does not mean you have to configure all. The line directly under the header explain it "_One_ of the following permissions is required".

Since the ReadWrite supersedes the Read permission, just the Read permission will also work.

Reply:
No problem. Will change it in the evening